components/eam/bld/namelist_files/use_cases/1850_eam_CMIP6_chemUCI-Linoz-mam5-vbs-1pctCO2.xml

```diff
@@ -76,10 +76,10 @@
 
 <!-- <tracer_cnst_filelist>this_field_is_not_used</tracer_cnst_filelist> -->
 <tracer_cnst_type    >CYCLICAL</tracer_cnst_type>
-<tracer_cnst_cycle_yr>1995</tracer_cnst_cycle_yr>
-<tracer_cnst_file    >ch4_oxid_1.9x2.5_L26_1990-1999clim.c090804.nc</tracer_cnst_file>
+<tracer_cnst_cycle_yr>1849</tracer_cnst_cycle_yr>
```

Comment:
Why is the default year set to 1849 that is not in the year range of input file?

Reply:
The tacer cnst file proposed to use only has 1849, 1855, 1865, ..., 1995, 2005, 2015. I just use 1849 following v1/v2 practice.